packages/discord.js/typings/index.d.ts

```diff
@@ -2837,17 +2836,18 @@ export class GuildChannelManager extends CachedManager<Snowflake, GuildBasedChan
   private constructor(guild: Guild, iterable?: Iterable<RawGuildChannelData>);
   public readonly channelCountWithoutThreads: number;
   public guild: Guild;
+
+  public create<T extends Exclude<GuildChannelTypes, ChannelType.GuildStore>>(
+    name: string,
+    options: GuildChannelCreateOptions & { type: T },
+  ): Promise<MappedGuildChannelTypes[T]>;
   /** @deprecated See [Self-serve Game Selling Deprecation](https://support-dev.discord.com/hc/en-us/articles/4414590563479) for more information */
   public create(
     name: string,
-    options: GuildChannelCreateOptions & { type: ChannelType.GuildStore },
+    options: GuildChannelCreateOptions & { type: ChannelType.GuildStore | 'GuildStore' },
```

Comment:
```suggestion
    options: GuildChannelCreateOptions & { type: ChannelType.GuildStore },
```

Reply:
Oof, managed to fix this before I saw this haha!